utils/index.d.ts

```diff
@@ -1685,8 +1690,15 @@ export declare enum AzExtResourceType {
 }
 
 /**
- * Describes command callbacks for tree node context menu commands
+ * Gets the exported API from the given extension id and version range.
+ * Automatically populates the consumer extension id to the created API.
+ *
+ * @param extensionId The extension id to get the API from
+ * @param apiVersionRange The version range of the API you need. Any semver syntax is allowed. For example "1" will return any "1.x.x" version or "1.2" will return any "1.2.x" version
+ * @throws Error if extension with id is not installed.
  */
+export function getExtensionApi<T extends AzureExtensionApi>(extensionId: string, apiVersionRange: string): Promise<T>;
```

Comment:
Is this meant to be this?
```suggestion
export declare function getAzureExtensionApi<T extends AzureExtensionApi>(extensionId: string, apiVersionRange: string): Promise<T>;
```

Reply:
Does adding the `declare` inside a `.d.ts` file make a difference?

Reply:
I don't think it makes a significant difference, but `declare` is more "correct".